packages/mcp/mcp-client-chat/src/mcp-client-chat.ts

```
@@ -187,14 +194,25 @@ export abstract class McpClientChat {
   protected async chatIteration(): Promise<void> {
     try {
       while (this.iterationSteps > 0) {
-        const response: ChatCompleteResponse | Error = await this.queryChatComplete();
+        let response: ChatCompleteResponse | Error;
+
+        if (this.streamSwitch) {
+          try {
+            const streamResponses: ReadableStream = await this.queryChatCompleteStreaming();
```

ESLint

[error] The 'ReadableStream' is still an experimental feature The configured version range is '^20.19.0 || >=22.12.0'.

(n/no-unsupported-features/node-builtins)